landscaper/gardenlet/pkg/apis/imports/v1alpha1/types.go

```diff
@@ -0,0 +1,59 @@
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"encoding/json"
+
+	landscaperv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	seedmanagementv1alpha1 "github.com/gardener/gardener/pkg/apis/seedmanagement/v1alpha1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Imports defines the landscaper import for the Gardenlet.
+type Imports struct {
+	metav1.TypeMeta `json:",inline"`
+	// SeedCluster contains the kubeconfig for the cluster
+	// - into which the Gardenlet is deployed by the landscaper
+	// - that is targeted as the Seed cluster by the Gardenlet via the default in-cluster mounted service account token
+	// Hence, the Gardenlet is always deployed into the Seed cluster itself.
+	// Deploying the Gardenlet outside of the Seed cluster is not supported (e.g., landscaper deploys Gardenlet into
+	// cluster A and the Gardenlet is configured via field .seedClientConnection.kubeconfig to target cluster B as Seed)
+	SeedCluster landscaperv1alpha1.Target `json:"seedCluster"`
+	// GardenCluster is the landscaper target containing the kubeconfig for the
+	// Garden cluster (sometimes referred to as "virtual garden" - with Gardener API groups!)
+	GardenCluster landscaperv1alpha1.Target `json:"gardenCluster"`
+	// SeedBackupCredentials contains the credentials for an optional backup provider for the Seed cluster registered by the Gardenlet
+	// required when the Seed is configured for Backup (configured in the Gardenlet component configuration in the field seedConfig.spec.backup).
+	// before deploying the Gardenlet, the landscaper deploys a secret containing the specified credentials into the Garden cluster
+	// +optional
+	SeedBackupCredentials *json.RawMessage `json:"seedBackupCredentials,omitempty"`
+	// DeploymentConfiguration configures the Kubernetes deployment of the Gardenlet
+	// +optional
+	DeploymentConfiguration *seedmanagementv1alpha1.GardenletDeployment `json:"deploymentConfiguration,omitempty"`
+	// ComponentConfiguration specifies values for the Gardenlet component configuration
+	// This results in the configuration file loaded at runtime by the deployed Gardenlet
+	ComponentConfiguration runtime.RawExtension `json:"componentConfiguration"`
+	// ImageVectorOverwrite contains an optional image vector override.
+	// +optional
+	ImageVectorOverwrite *json.RawMessage `json:"imageVectorOverwrite,omitempty" protobuf:"bytes,12,opt,name=imageVectorOverwrite"`
```

Comment:
You can remove the protobuf tags as currently there is no protobuf generation requested/configured for this pkg. I assume that is is just a copy-paste leftover.

```suggestion
	ImageVectorOverwrite *json.RawMessage `json:"imageVectorOverwrite,omitempty"`
```